src/event/behavior/keydown.ts

```diff
@@ -28,7 +28,7 @@ const keydownBehavior: {
   ArrowDown: (event, target, instance) => {
     /* istanbul ignore else */
     if (isElementType(target, 'input', {type: 'radio'} as const)) {
-      return () => walkRadio(instance, target, -1)
+      return () => walkRadio(instance, target, 1)
```

Comment:
As [the W3C specification about radio group](https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/#kbd_label) describes, pressing the Down Arrow key behaves the same way as the Right Arrow key. "Moves focus to and checks the next radio button in the group" when the Down/Right arrow pressed, so the `direction` of `walkRadio` should be 1.

The same is true for the Up and Left Arrow keys, so the `direction` should be -1.